test/pipeline/pipeline.import.azure.master.json

```diff
@@ -630,7 +1117,46 @@
                             "showWarnings": "false",
                             "ignoreLASTEXITCODE": "false",
                             "pwsh": "true",
-                            "workingDirectory": ""
+                            "workingDirectory": "",
+                            "runScriptInSeparateScope": "false"
+                        }
+                    },
+                    {
```

Comment:
bicep errors label add/remove

Reply:
What's the intent here?  Thinking that a bicep build error should fail the PR... so a label would only be needed if we treat them as "warnings"...

Reply:
"bicep warnings"